packages/bpk-component-sliding-drawer/src/BpkSlidingDrawerContent.js

```diff
@@ -0,0 +1,122 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2017 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import Transition from 'react-transition-group/Transition';
+import { animations } from 'bpk-tokens/tokens/base.es6';
+import { cssModules } from 'bpk-react-utils';
+import { BpkButtonLink } from 'bpk-component-link';
+import BpkCloseButton from 'bpk-component-close-button';
+
+import STYLES from './bpk-sliding-drawer-content.scss';
+
+const getClassName = cssModules(STYLES);
+
+const BpkSlidingDrawerContent = (props) => {
+  const contentClassNames = [getClassName('bpk-sliding-drawer')];
```

Comment:
Convention is to first establish all the class name arrays and then do the conditional classes i.e.:
```
const contentClassNames = [getClassName('bpk-sliding-drawer')];
const headerClassNames = [getClassName('bpk-sliding-drawer__heading')];

if (props.className) { contentClassNames.push(props.className); }
if (props.hideTitle) { headerClassNames.push(getClassName('bpk-sliding-drawer__heading--visually-hidden')); }
```

Reply:
why? that's just harder to follow